docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1098,6 +1099,144 @@ team can more easilly read the messages.
 It is also possible to configure the persistence properties for a specific tenant, for example to set an alternate
 database hostname or database schema name.
 
+## 4.10 Bulk data operations
+### 4.10.1 Bulk data export
+Bulk data export is implemented according to the [HL7 FHIR BulkDataAccess IG: STU1](http://hl7.org/fhir/uv/bulkdata/STU1/export/index.html).
+There are 2 modules involved inside the implementation:
+- fhir-operation-bulkdata
+- fhir-bulkimportexport-webapp   
+
+To integration test, there are tests in ExportOperationTest.java in fhir-server-test module with server integration test cases for system, patient and group export.  
```

Comment:
```suggestion
```
I think this should be removed.  I don't think it's necessary, maybe include as a readme in the bulkdata operation module.